spec/DatabaseController.spec.js

```diff
@@ -1,9 +1,10 @@
+const Config = require('../lib/Config');
 const DatabaseController = require('../lib/Controllers/DatabaseController.js');
 const validateQuery = DatabaseController._validateQuery;
 
-describe('DatabaseController', function () {
-  describe('validateQuery', function () {
-    it('should not restructure simple cases of SERVER-13732', done => {
+describe('DatabaseController', () => {
```

Comment:
Are these changes related to this PR? If not, please revert. They are all over the PR which makes it difficult to review.